modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/MessageManagementInformationMBean.java

```diff
@@ -99,26 +99,26 @@ public String getObjectInstanceName() {
             throws MBeanException{
         List<CompositeData> compositeDataList = new ArrayList<>();
 
-        try {
-            ProtocolType protocolType = new ProtocolType(protocolTypeAsString);
-
-            List<AndesMessageMetadata> nextNMessageMetadataFromQueue;
-            if (!DLCQueueUtils.isDeadLetterQueue(destinationName)) {
-                nextNMessageMetadataFromQueue = Andes.getInstance()
-                        .getNextNMessageMetadataFromQueue(destinationName, nextMessageID, limit);
-            } else {
-                nextNMessageMetadataFromQueue = Andes.getInstance()
-                        .getNextNMessageMetadataFromDLC(destinationName, 0, limit);
-            }
-
-            for (AndesMessageMetadata andesMessageMetadata : nextNMessageMetadataFromQueue) {
-                compositeDataList.add(messagesCompositeDataHelper.getMessageAsCompositeData(protocolType,
-                                                                                        andesMessageMetadata, content));
-            }
-
-        } catch (AndesException | OpenDataException e) {
-            throw new MBeanException(e, "Error occurred in browse queue.");
-        }
+//        try {
```

Comment:
Commented because of compilation error. Next step is to remove the these qpid classes.